templates/BenchmarkDotNet.BenchmarkProjectTemplate.FSharp/BenchmarkDotNet.BenchmarkProjectTemplate.FSharp.nuspec

```diff
@@ -9,7 +9,6 @@
     <tags>benchmark;benchmarking;performance;template</tags>
     <iconUrl>https://raw.githubusercontent.com/dotnet/BenchmarkDotNet/master/docs/guide/logo.png</iconUrl>
     <projectUrl>https://benchmarkdotnet.org</projectUrl>
-    <license type="expression">MIT</license>
```

Comment:
I couldn't get the package to create without removing this

Reply:
Which nuget version did you use?

Reply:
NuGet Version: 4.8.2.5835

Reply:
Not sure what we should do about that - last recommended [version is 5.0.2](https://www.nuget.org/downloads).

Reply:
This was version mono 5.18 installed on my machine, 5.20 is out so I'll download that and try it out.

Reply:
Oh I had to use `nuget update -self`. And I just reverted this.